src/notation/internal/notationinteraction.cpp

```diff
@@ -7339,6 +7339,57 @@ void NotationInteraction::addMelisma()
     toLyrics->selectAll(toLyrics->cursor());
 }
 
+void NotationInteraction::addElision() {
+
+    if (!m_editData.element || !m_editData.element->isLyrics()) {
```

Comment:
I'd change the condition to `!isTextEditingStarted()`:
- the elision shortcut may be useful in other kinds of text (other than lyrics) as well
- `isTextEditingStarted` guarantees that the edited element is really in text edit mode